misc/services/camel/de-metas-camel-externalsystems/de-metas-camel-grssignum/src/main/java/de/metas/camel/externalsystems/grssignum/to_grs/api/model/JsonBPartner.java

```diff
@@ -103,10 +109,13 @@ public JsonBPartner(
 			@JsonProperty("ORT") final @Nullable String city,
 			@JsonProperty("LANDESCODE") final @Nullable String countryCode,
 			@JsonProperty("GLN") final @Nullable String gln,
-			@JsonProperty("KONTAKTE") final @Nullable @Singular List<JsonBPartnerContact> contacts)
+			@JsonProperty("METASFRESHID") final @Nullable String metasfreshId,
```

Comment:
again, ok to be nullable, but we'll need to later throw a specific error, because functionally it may not be null